src/citrine/_serialization/serializable.py

```diff
@@ -3,6 +3,8 @@
 
 Self = TypeVar('Self', bound='Serializable')
 
+CITRINE_SCOPE = 'id'
+
```

Comment:
This object was chosen for where to define CITRINE_SCOPE because most other reasonable places would cause circular dependencies.  I'm not convinced it's the best place, but there is no clearer better place.

Reply:
What about [here](https://github.com/CitrineInformatics/citrine-python/blob/643e1f49814622a5bea668413e4295d5d7d2a9a9/src/citrine/resources/data_concepts.py#L19)

Reply:
I missed one; nice catch.  Data Concepts would be a more elegant place for it to live, but `_utils.functions` gets imported there and `_utils.functions` needs the constant, too.  That's the spoiler on circularity.

Reply:
only `_utils.functions.get_object_id` needs it, so you could move the import into that function body to break the cycle.  That function is not imported by data_concepts.py

Reply:
I was just about to double check that.  I'll buy that reasoning.  Update in flight.